config/routes.rb

```diff
@@ -190,11 +190,13 @@
     end
   end
 
-  resources :smart_proxies, :except => [:show] do
+  resources :smart_proxies do
     member do
       post 'ping'
       put 'refresh'
       get 'version'
+      post 'plugin_version'
```

Comment:
but you are expecting to GET the data right? a more proper HTTP usage might be:
GET plugin_version?name=...